### PR TITLE
check for pytest.ini file

### PR DESCRIPTION
**Pull request by kshychko**
to run unit tests with `pytest` instead of `manage.py test`

### CODE REVIEW

File: jenkins/aws/buildPython.sh [hunk 1]
```diff
@@ -69,25 +69,33 @@ function main() {
...
-      ENV_FILE=${PYTHON_UNIT_TEST_ENV_FILE} python manage.py test ${MANAGE_OPTIONS} ||
+      python pytest ${MANAGE_OPTIONS} ||
         { exit_status=$?; fatal "Tests failed"; return ${exit_status}; }
     else
-      warning "No manage.py - no tests run"
+      if [[ -f manage.py ]]; then
+        info "Running unit tests ..."
+        MANAGE_OPTIONS=""
+        if [[ -n ${TEST_REPORTS_DIR} ]]; then
+          if [[ -n ${TEST_JUNIT_DIR} ]]; then
```
Comment (roleyfoley) [inline jenkins/aws/buildPython.sh]: Should  Junit also apply to pytest? does it provide a test results file for jenkins?

Comment (kshychko) [reply jenkins/aws/buildPython.sh]: @roleyfoley added within next commit

File: jenkins/aws/buildPython.sh [hunk 2]
```diff
@@ -69,25 +69,40 @@ function main() {
...
         MANAGE_OPTIONS+=" ${UNIT_OPTIONS}"
       fi
-
-      ENV_FILE=${PYTHON_UNIT_TEST_ENV_FILE} python manage.py test ${MANAGE_OPTIONS} ||
+      python pytest ${MANAGE_OPTIONS} ||
         { exit_status=$?; fatal "Tests failed"; return ${exit_status}; }
     else
-      warning "No manage.py - no tests run"
+      if [[ -f manage.py ]]; then
+        info "Running unit tests ..."
```
Comment (ml019) [inline jenkins/aws/buildPython.sh]: maybe just align the info messages ...   "with manage.py"

Comment (kshychko) [reply jenkins/aws/buildPython.sh]: @ml019 done in 2c53f98

**Pull request merged**